recipes/libxml/libxml2.inc

```diff
@@ -11,18 +11,39 @@ DEPENDS_HOST_OS = "libdl"
 DEPENDS_HOST_OS:HOST_LIBC_mingw = ""
 DEPENDS_${PN} += "libc libm ${DEPENDS_HOST_OS}"
 
+EXTRA_OECONF_PYTHON = "--with-python-install-dir=${libdir}"
```

Comment:
I guess it uses the hosts python2 :-)
And it would be rather difficult to bake using oe-lite without native python2.